dali/pipeline/operators/reader/parser/tf_feature.h

```diff
@@ -58,9 +66,13 @@ class Feature {
 
   FeatureType GetType() const { return type_; }
   bool HasShape() const { return has_shape_; }
+  bool HasPartialShape() const { return has_partial_shape_; }
   const std::vector<Index>& Shape() const {
     return shape_;
   }
+  std::vector<Index> PartialShape() const {
```

Comment:
Why not return the reference?

Reply:
OK, I see now.

Reply:
I modify it outside (add first dimension) and use to resize output buffer. But maybe it would be better  to return reference here and copy it into new vector outside?